### ISSUE TITLE
[Prompt] Pasting Clipboard into Prompt Secret Returns Corrupted Data

### ISSUE THREAD

**Issue by nakleiderer** [c4spar/deno-cliffy#54]
First of all, this is an awesome project, and I've been enjoying using it! 

While implementing an authentication CLI command, I noticed the value of the password that I was pasting in the console was not being captured correctly into the variable.

# Bug Description
When pasting into the console, the value pasted is not equal to the value capture by Cliffy Prompt.

# Reproduction Steps
- Create a script.ts file with the following code:

```ts
// script.ts
import { Secret } from "https://raw.githubusercontent.com/c4spar/deno-cliffy/v0.11.2/prompt.ts";

const password = await Secret.prompt({message:"Password:",hidden:true})
console.log(password)
```
- Copy `0123456789abcdefghijklmnopqrstuvwxyz` to your clipboard.
- Run the script: `deno run --unstable script.ts`
- When prompted for password, paste the clipboard into the console.
- Expected output: `0123456789abcdefghijklmnopqrstuvwxyz`
- Actual Output: `08gow`

# Environment information
- Deno Version: deno 1.2.1
- Cliffy Version: 0.11.2
- OS: macOS Catalina v10.15.5
- Shell: zsh 5.7.1 (x86_64-apple-darwin19.0)

**Comment by c4spar** [comment]
Hi @nakleiderer,
Thx for using cliffy and your issued! I will look into it.

**Comment by c4spar** [comment]
@nakleiderer i have found the problem and fixed in v0.12.0.

The problem was, the prompt module reads always 8-bit's at once into an `UInt8Array`, processes the data and reads the next 8-bit's.
So, `0123456789abcdefghijklmnopqrstuvwxyz` will be read as `01234567` `89abcdef` `ghijklmn` `opqrstuv` `wxyz`.
But the keycode module has only captured the first character of each `UInt8Array`, so the result was: `08gow`.

The issue is fixed and the keycode and prompt modules should capture all characters now.
